scripts/sync_milestones.js

```diff
@@ -12,6 +12,7 @@ var ghlib = require('./ghlib.js')
 var P = require('bluebird')
 
 var ACTIVE_FEATURE_LABELS = {
+  "designin": true,
```

Comment:
Sorry for the belated drive-by but it looks like `"designing"` is missing the trailing `g` here.